src/main/java/frc/robot/commands/AutoChargeStationOneConeOtherSide.java

```diff
@@ -4,40 +4,48 @@
 
 package frc.robot.commands;
 
-
 import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitCommand;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+
+import frc.robot.FieldConstants;
 import frc.robot.Constants.Position;
 import frc.robot.subsystems.Arm;
 import frc.robot.subsystems.Claw;
 import frc.robot.subsystems.DriveTrain;
 import frc.robot.subsystems.Vision;
 
 public class AutoChargeStationOneConeOtherSide extends SequentialCommandGroup implements AutoCommandInterface {
-
-    // private final double BACKING_MPS = 0.5;
-
-    AutoFollowTrajectory m_traj;
+    final Pose2d m_initialPoseBlue;
 
     /** Creates a new AutoChargeStationOneCube */
-    public AutoChargeStationOneConeOtherSide(DriveTrain driveTrain, Arm arm, Vision vision, Claw claw, JoystickButton overrideButton) {
-        // Note this is a quick hack: the trajectory is loaded to just get the initial Pose and the stop point for backing up.
-        //  Otherwise it is not actually used.
-        m_traj = new AutoFollowTrajectory(driveTrain, "c_out_the_zone_balance");
+    public AutoChargeStationOneConeOtherSide(DriveTrain driveTrain, Arm arm, Vision vision, Claw claw, boolean wallPosition, JoystickButton overrideButton) {
+        // barrier cone position
+        double initY = FieldConstants.CHARGE_STATION_CENTER_Y + Units.inchesToMeters(22.0);
+        if (wallPosition) {
+            // wall cone position
+            initY = FieldConstants.CHARGE_STATION_CENTER_Y - Units.inchesToMeters(22.0);
+        }
+
+        m_initialPoseBlue = new Pose2d(2.0, initY, Rotation2d.fromDegrees(180));
```

Comment:
the starting X was 2.2 in the path planner. Are we tuning this? This should be > 2 because otherwise the arm won't stow

Reply:
Yes, we pulled it in during shed testing yesterday.